pkgs/shells/zsh/zsh-autopair/default.nix

```diff
@@ -2,17 +2,15 @@
 
 stdenv.mkDerivation rec {
   pname = "zsh-autopair";
-  version = "1.0";
+  version = "2024-07-14";
```

Comment:
```suggestion
  version = "1.0-unstable-2024-07-14";
```

Reply:
@pbsds Why would we set it to unstable ? No tags where released since the v1.0 and commits are still being created. It would be kinda awkward to see a unstable version where the repository itself doesn't "tag" it as so. IMO

Reply:
the `unstable` schema is IMO misleading in many cases, but this is the standard schema nixpkgs has settled on. We keep the last applicable release as a prefix to communicate it is effectively 1.0++.

Reply:
you can see on [repology](https://repology.org/project/zsh-autopair/versions) that aur for example suffixes the current git revision

Reply:
you could consider communicating with upstream about cutting a new release, if you do not like the `unstable` connotation

Reply:
@pbsds Done ^^